core/src/modules/ops/runs.ts

```diff
@@ -144,9 +144,10 @@ export namespace RunOps {
 
       // there are 3 phases to group runs, but only 2 really could have work, so we attribute 1/2 to each phase
       let percentComplete = Math.floor(
-        100 *
-          (membersAlreadyUpdated /
-            (totalGroupMembers > 0 ? totalGroupMembers : 1))
+        membersAlreadyUpdated === 0
+          ? 1
+          : membersAlreadyUpdated /
```

Comment:
It looks like this was a percentage (out of 100) before, but it's no longer being multiplied by 100. I believe the `100 *` should still apply?

Reply:
Good catch! Fixed via aa13824e4a36fec8689354da6b2e271362a936fc